monochart/templates/deployment.yaml

```diff
@@ -62,6 +62,9 @@ spec:
 
     spec:
       serviceAccountName: {{ template "monochart.serviceAccountName" . }}
+      {{- if .Values.deployment.pod.hostPID }}
+      hostPID: true
+      {{- end }}
```

Comment:
Or in this case, is it actually evaluating the boolean, in which case, if hostPID isn't set, would it fail as the key doesn't exist?

Reply:
> Would this lead to a situation where somebody explicitly sets hostPID: false and since the key exists, it will set it to true anyways?

Using the variable was the first approach I tried, but kubernetes complained that it wanted a boolean value.

I tested with it set to `false` in my values file as well:

```yaml
deployment:
  enabled: true
  pod:
    hostPID: false
    annotations:
      cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
```